src/components/Column/Column.test.tsx

```diff
@@ -35,6 +36,36 @@ describe("<Column />", () => {
     fireEvent.blur(renderedEditableLabel);
 
     // Assert
-    expect(mockedUseMutationCall).toBeCalled();
+    expect(mockedUseMutationCall).toBeCalledTimes(1);
+    expect(mockedUseMutationCall).toBeCalledWith({
+      variables: { name: changedMockedValue },
+    });
+  });
+
+  test("passed appropriate props - it should render given ticket with title", () => {
```

Comment:
This text description is exactly the same as the one above. What did you really want to test here?
I think firing events to see the behavior or your custom hook is a bit too much unless you give comments on them. I would make a separate case for the event check for visibility.
It comes to a preference though, it's  an example but you can also just leave comments on an appropriate place like;

```typescript
    // If label has been edited and keyboard got blured, it should treggier call with useMutation
    expect(mockedUseMutationCall).toBeCalledWith({
      variables: { name: changedMockedValue },
    });
```